orttraining/tools/ci_test/download_azure_blob.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
You need this file?

Reply:
We're going to need to recreate this PR/force-push with a cherry-pick, as it contains unrelated changes, and `DmlOperatorQLinearAdd` which should be excluded, because 1.5.1 does not support that function.